--- openmw-openmw-0.29.0.or/components/files/linuxpath.cpp	2014-03-04 08:52:48.000000000 +0100
+++ openmw-openmw-0.29.0/components/files/linuxpath.cpp	2014-03-13 19:31:40.400005365 +0100
@@ -80,7 +80,7 @@
 
 boost::filesystem::path LinuxPath::getGlobalDataPath() const
 {
-    boost::filesystem::path globalDataPath("/usr/share/games/");
+    boost::filesystem::path globalDataPath("/usr/share/");
     return globalDataPath / mName;
 }
 
