diff -ru openmw-openmw-0.36.1.orig/plugins/mygui_resource_plugin/CMakeLists.txt openmw-openmw-0.36.1/plugins/mygui_resource_plugin/CMakeLists.txt
--- plugins/mygui_resource_plugin/CMakeLists.txt	2015-05-31 14:21:21.000000000 +0200
+++ plugins/mygui_resource_plugin/CMakeLists.txt	2015-08-05 14:39:10.010972034 +0200
@@ -8,7 +8,7 @@
     Plugin_MyGUI_OpenMW_Resources
 )
 
-add_definitions("-D_USRDLL -DMYGUI_BUILD_DLL")
+add_definitions("-D_USRDLL -DMYGUI_BUILD_DLL -fPIC")
 
 add_library(${MYGUI_RESOURCE_PLUGIN_LIBRARY}
     SHARED
