diff -Naur openmw-0.25.0.orig/CMakeLists.txt openmw-0.25.0/CMakeLists.txt
--- openmw-0.25.0.orig/CMakeLists.txt	2013-09-10 13:47:37.216378534 -0500
+++ openmw-0.25.0/CMakeLists.txt	2013-09-10 15:42:05.760450935 -0500
@@ -166,6 +166,19 @@
     message(WARNING "--------------------")
 endif (NOT FFMPEG_FOUND)
 
+# TinyXML
+option(USE_SYSTEM_TINYXML "Use system TinyXML library instead of internal." OFF)
+if(USE_SYSTEM_TINYXML)
+    find_library(TINYXML_LIBRARIES tinyxml)
+    find_path(TINYXML_INCLUDE_DIR tinyxml.h)
+    message(STATUS "Found TinyXML: ${TINYXML_LIBRARIES} ${TINYXML_INCLUDE_DIR}")
+    if(TINYXML_LIBRARIES AND TINYXML_INCLUDE_DIR)
+        include_directories(${TINYXML_INCLUDE_DIR})
+        message(STATUS "Using system TinyXML library.")
+    else()
+        message(FATAL_ERROR "Detection of system TinyXML incomplete.")
+    endif()
+endif()
 
 # Platform specific
 if (WIN32)
diff -Naur openmw-0.25.0.orig/extern/oics/CMakeLists.txt openmw-0.25.0/extern/oics/CMakeLists.txt
--- openmw-0.25.0.orig/extern/oics/CMakeLists.txt	2013-07-26 07:02:41.000000000 -0500
+++ openmw-0.25.0/extern/oics/CMakeLists.txt	2013-09-10 15:18:15.652360189 -0500
@@ -9,12 +9,23 @@
 	ICSInputControlSystem_keyboard.cpp
 	ICSInputControlSystem_mouse.cpp
 	ICSInputControlSystem_joystick.cpp
+)
+
+set(TINYXML_SOURCE_FILES
 	tinyxml.cpp
 	tinyxmlparser.cpp
 	tinyxmlerror.cpp
-    tinystr.cpp
+	tinystr.cpp	
 )
 
-add_library(${OICS_LIBRARY} STATIC ${OICS_SOURCE_FILES})
+if(USE_SYSTEM_TINYXML)
+    add_library(${OICS_LIBRARY} STATIC ${OICS_SOURCE_FILES})
+    target_link_libraries(${OICS_LIBRARY} ${TINYXML_LIBRARIES})
+else()
+    add_library(${OICS_LIBRARY} STATIC
+    ${OICS_SOURCE_FILES}
+    ${TINYXML_SOURCE_FILES})
+endif()
 
+# Does this do anything?
 link_directories(${CMAKE_CURRENT_BINARY_DIR})
