diff -up openmw-openmw-0.37.0/CMakeLists.txt.ffmpeg openmw-openmw-0.37.0/CMakeLists.txt
--- openmw-openmw-0.37.0/CMakeLists.txt.ffmpeg	2015-11-22 19:54:17.000000000 +0100
+++ openmw-openmw-0.37.0/CMakeLists.txt	2015-12-09 10:35:07.601102581 +0100
@@ -108,6 +108,7 @@ unset(FFMPEG_LIBRARIES CACHE)
 find_package(FFmpeg REQUIRED)
 
 set (FFMPEG_LIBRARIES ${FFMPEG_LIBRARIES} ${SWSCALE_LIBRARY} ${SWRESAMPLE_LIBRARIES})
+include_directories(${FFMPEG_INCLUDE_DIRS})
 
 if ( NOT AVCODEC_FOUND OR NOT AVFORMAT_FOUND OR NOT AVUTIL_FOUND OR NOT SWSCALE_FOUND OR NOT SWRESAMPLE_FOUND)
     message(FATAL_ERROR "FFmpeg component required, but not found!")
