--- openmw-openmw-0.36.1.orig/plugins/mygui_resource_plugin/CMakeLists.txt	2015-05-31 14:21:21.000000000 +0200
+++ openmw-openmw-0.36.1/plugins/mygui_resource_plugin/CMakeLists.txt	2015-08-05 14:58:23.061338391 +0200
@@ -9,6 +9,7 @@
 )
 
 add_definitions("-D_USRDLL -DMYGUI_BUILD_DLL")
+set_target_properties(${MYGUI_RESOURCE_PLUGIN_LIBRARY} PROPERTY POSITION_INDEPENDENT_CODE TRUE)
 
 add_library(${MYGUI_RESOURCE_PLUGIN_LIBRARY}
     SHARED
