--- openmw-0.28.0-source/files/openmw.desktop	2014-01-15 13:15:40.571392721 +0100
+++ openmw-0.28.0-source-orig/files/openmw.desktop	2014-01-08 20:27:33.000000000 +0100
@@ -3,7 +3,7 @@
 Name=OpenMW Launcher
 GenericName=Role Playing Game
 Comment=An engine replacement for The Elder Scrolls III: Morrowind
-Keywords=Morrowind;Reimplementation Mods;esm;bsa
+Keywords=Morrowind;Reimplementation Mods;esm;bsa;
 TryExec=omwlauncher
 Exec=omwlauncher
 Icon=openmw
